src/Components/Login/Login.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../scss/variables';
+@import '../../scss/mixins';
+
+form {
```

Comment:
Should we nest these with scss?

Reply:
Do you mean the imports?  Not sure what you mean here.

Reply:
In regards to each css class. It's not a big deal but could make it a little easier to read.

Something like this... 

<img width="721" alt="Screen Shot 2020-01-05 at 12 06 19 PM" src="https://user-images.githubusercontent.com/48968224/71784721-c9d43300-2fb3-11ea-8f07-b0e532018e39.png">